src/JITModule.cpp

```diff
@@ -334,7 +334,12 @@ void JITModule::compile_module(std::unique_ptr<llvm::Module> m, const string &fu
     // Build TargetMachine
     llvm::orc::JITTargetMachineBuilder tm_builder(llvm::Triple(m->getTargetTriple()));
     tm_builder.setOptions(options);
+#if LLVM_VERSION >= 180
+    tm_builder.setCodeGenOptLevel(CodeGenOptLevel::Aggressive);
+#else
     tm_builder.setCodeGenOptLevel(CodeGenOpt::Aggressive);
+#endif
+    tm_builder.setCodeGenOptLevel(opt_level);
```

Comment:
I'm confused by this change. It calls tm_builder.setCodeGenOptLevel twice?

Reply:
Bah, bonehead copy/paste error